app/features/navbar/components/HelpAction.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+
+import HelpIcon from '@atlaskit/icon/glyph/question-circle';
+import Droplist, { Item } from '@atlaskit/droplist';
+
+import React, { Component } from 'react';
+
+import config from '../../config';
+import { openExternalLink } from '../../utils';
+
+type State = {
+
+    /**
+     * Whether the droplist is open or not.
+     */
+    droplistOpen: boolean
+};
+
+/**
+ * Help Action for Navigation Bar.
+ */
+class HelpAction extends Component< *, State> {
+    /**
+     * Initializes a new {@code HelpAction} instance.
+     *
+     * @inheritdoc
+     */
+    constructor() {
+        super();
+
+        this.state = {
+            droplistOpen: false
+        };
+
+        this._droplistToggle = this._droplistToggle.bind(this);
+        this._openTermsPage = this._openTermsPage.bind(this);
+        this._openPrivacyPage = this._openPrivacyPage.bind(this);
+    }
+
+    _droplistToggle: (*) => void;
+
+    /**
+     * Toggles the droplist.
+     */
+    _droplistToggle() {
+        this.setState({
+            droplistOpen: !this.state.droplistOpen
+        });
+    }
+
+    _openTermsPage: (*) => void;
+
+    /**
+     * Opens Terms and Conditions Page in default browser.
+     */
+    _openTermsPage() {
+        openExternalLink(config.termsAndConditionsURL);
+    }
+
+    _openPrivacyPage: (*) => void;
+
+    /**
+     * Opens Privacy Policy Page in default browser.
+     */
+    _openPrivacyPage() {
+        openExternalLink(config.privacyPolicyURL);
+    }
+
+    /**
+     * Render function of component.
+     *
+     * @return {ReactElement}
+     */
+    render() {
+        return (
+            <Droplist
+                isOpen = { this.state.droplistOpen }
+                onClick = { () => this._droplistToggle() }
+                onOpenChange = { () => this._droplistToggle() }
+                position = 'right bottom'
+                trigger = { <HelpIcon /> } >
+                <Item onActivate = { this._openTermsPage }>Terms</Item>
+                <Item onActivate = { this._openPrivacyPage }>Privacy</Item>
```

Comment:
can we please add a support link here? it should point to `mailto:support@jitsi.org`

Reply:
Yeah. Sure.

Reply:
As `support` or `feedback`?

Reply:
Right. Ler's go with "Send Feedback" since we have that on mobile too.

Reply:
Done.